release/_releases_json.sh

```diff
@@ -31,6 +41,26 @@ function _merge_json_snippets {
 	fi
 }
 
+function _process_new_product {
+	local releases_json="${_RELEASE_ROOT_DIR}/release-data/promotion/files/0000-00-00-releases.json"
```

Comment:
@Wykenz, don't we already have a variable holding the promotions directory? Please see https://github.com/liferay/liferay-docker/blob/9636f482b19421e4e120649fe11a29991caa79e7/release/release_gold.sh#L39. Can we use it here?